unix-training-notebooks/Answers.ipynb

```diff
@@ -182,23 +182,27 @@
     "  \n",
     "`awk -F\"\\t\" '$4 ~ /gene/ {print $4}' exercises.bed | sort -u | wc -l`  \n",
     "  \n",
-    "Alternatively, the names can be found like this:  \n",
+    "Counting how many times each of the unique names appear can be found like this:  \n",
     "  \n",
-    "`awk -F\"\\t\" '$4 ~ /gene/ {print $4}' exercises.bed  | sort | uniq -c | awk '$1>1'`  \n",
+    "`awk -F\"\\t\" '$4 ~ /gene/ {print $4}' exercises.bed  | sort | uniq -c ` \n",
+    "\n",
+    "Finding the counts that are greater than one (so the name is repeated) can be filtered like this \n",
```

Comment:
"Counts that are greater than one (so the name is repeated) can be filtered like this"

?